packages/core/src/browser/theming.ts

```diff
@@ -50,10 +50,7 @@ export class ThemeService {
         return global[ThemeServiceSymbol] || new ThemeService();
     }
 
-    protected constructor(
-        protected _defaultTheme: string | undefined = FrontendApplicationConfigProvider.get().defaultTheme,
-        protected fallbackTheme: string = 'dark'
-    ) {
+    protected constructor() {
```

Comment:
This is a breaking change. Please mention in the changelog.

Reply:
I don't think it is feasible to create another instance of `ThemingSerivce`. It is completely outside of DI. Do you know how could it happen?

Reply:
OK, leave it out. Thanks for checking!